content/docs/guides/components/chainer.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Chainer Training"
+description = "Instructions for using Chainer for training"
+weight = 10
```

Comment:
Can you adjust the weight so that the components section is alphabetized by component name:
You can check the preview
https://deploy-preview-200--eager-nobel-1708cf.netlify.com/docs/about/kubeflow/

Reply:
that looks reasonable.